src/components/PatientRegistration/PatientRegistration.tsx

```diff
@@ -22,6 +25,14 @@ export default function PatientRegistration(): ReactElement {
   const [consentMinorChild, setConsentMinorChild] = useState('');
   const [relationshipChild, setRelationshipChild] = useState('');
 
+  const [conditionsError, setConditionsError] = useState('');
+  const [followingError, setFollowingError] = useState('');
+  const [stressfulLevelError, setStressfulLevelError] = useState('');
+  const [phoneError, setPhoneError] = useState('');
+
+  const [emailError, setEmailError] = useState('');
+  const [medicationsError, setMedicationsError] = useState('');
+
 
   const itemsConditions = [
```

Comment:
This array is a constant. If you keep it inside of component, it will be recreated each time the component re-render cycle is called (even if the component is not re-rendered itself). It would be better to extract this array outside of component function body (preferably to some other file, to make this one smaller)

Reply:
Same for itemsFollowing

Reply:
thanks for your advice, it is very valuable to me! )